docs/recipes/MaestroSetup.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Maestro Setup
+description: Setting up e2e testing with Maestro in Ignite
+tags:
+  - Maestro
+  - testing
+last_update:
+  author: Dan Edwards
+publish_date: 2023-01-06
+---
+
+# Setting Up Maestro in Ignite
+
+## Overview
+
+E2E testing is a critical part of any application but it can be difficult to set up and maintain. Maestro is a tool that promises to be easy to set up and maintain e2e tests. This recipe will walk you through setting up Maestro in your Ignite project.
```

Comment:
E2E and e2e here and lowercase in the title. We might want to make this consistent throughout.

Also let's consider defining it at least once for those who don't know what it stands for.